readme.txt

```diff
@@ -26,13 +26,18 @@ The whole installation and configuration process is described in our [integratio
 1. Plugin configuration for gateway access
 2. Payment terminal configuration
 
+== Code Analysis==
+
+In order to test the code quality, phpstan config file is available. To run the analysis: 
+1. First install composer packages using 'composer install'
+2. Then run 'vendor/bin/phpstan analyze' to run the analysis. It'll print out any errors detected by PHPStan.
```

Comment:
This is missing some custom steps, if following the instructions as is it simply gives the following error:
```
plugin-wordpress$ vendor/bin/phpstan analyze
Note: Using configuration file /home/ajenbo/code/altapay/plugin-wordpress/phpstan.neon.dist.
Scanned directory /home/ajenbo/code/altapay/woocommerce does not exist.
```

Reply:
added additional information